test/docs/jdk/javadoc/doccheck/ExtLinksJdk.txt

```diff
@@ -261,9 +263,10 @@ https://docs.oracle.com/en/java/javase/22/docs/specs/man/java.html
 https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/lang/Double.html
 https://docs.oracle.com/en/java/javase/23/docs/api/java.base/java/math/BigDecimal.html
 https://docs.oracle.com/en/java/javase/23/docs/specs/man/java.html
-https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/lang/String.html
-https://docs.oracle.com/en/java/javase/24/docs/specs/javadoc/javadoc-search-spec.html
-https://docs.oracle.com/en/java/javase/24/docs/specs/man/javadoc.html
+https://docs.oracle.com/en/java/javase/24/docs/specs/man/java.html
```

Comment:
Is this right that some links are left pointing to version 23 and 24 resources, while most are updated to 25? 
I'm afraid it will be unpractical to manually update these links twice a year, so we should use some placeholder/macro to insert the current feature release. But that will only work if the links are also generated uniformly with the current feature release (for example by the `@extLink` taglet).

Reply:
The [java man page ](https://docs.oracle.com/en/java/javase/21/docs/specs/man/java.html#removed-java-options) has links to previous releases (I think these are added manually).

`SourceVersion.java` and `ClassFileFormatVersion.java` have hardcoded links to jls24 and jvms24 links, as in:

```
     * @see <a
     * href="https://docs.oracle.com/javase/specs/jls/se24/html/index.html">
     * <cite>The Java Language Specification, Java SE 24 Edition</cite></a>
     */
    RELEASE_24,
    ```

So I think anything that isn't pointing to `se25` is hardcoded to point to a different link, I could use a special macro for the current `se25` links (something like `@VERSION@` and then substitute it when reading from the file).